### PR TITLE
[enhance](mtmv)when calculating the availability of MTMV, no longer consider refresh state

### PR DESCRIPTION
**Pull request by zddr**
Before the modification, if the last refresh of the materialized view failed, the entire materialized view cannot be used for transparent rewriting. In fact, many partitions may be available, so this modification was made

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVRewriteUtil.java [hunk 1]
```diff
@@ -62,8 +61,7 @@ public static Collection<Partition> getMTMVCanRewritePartitions(MTMV mtmv, Conne
             return res;
         }
         // check mv is normal
-        if (!(mtmv.getStatus().getState() == MTMVState.NORMAL
-                && mtmv.getStatus().getRefreshState() == MTMVRefreshState.SUCCESS)) {
+        if (mtmv.getStatus().getState() != MTMVState.NORMAL) {
```
Comment (seawinde) [inline fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVRewriteUtil.java:64]: MTMVState#INIT means in building? May  MTMVState#INIT should not return

Comment (seawinde) [reply fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVRewriteUtil.java:64]: mtmv.getStatus().getState() == MTMVState.NORMAL && mtmv.getStatus().getRefreshState() !=  MTMVRefreshState.INIT maybe better

**Comment by zddr** [comment]
run buildall

**Review by seawinde — APPROVED** [review]

**Review by morrySnow — APPROVED** [review]

**Pull request merged**